README.md

```diff
@@ -1,118 +1,81 @@
+# Nuxeo Platform
 
+[![Build Status](https://jenkins.platform.dev.nuxeo.com/buildStatus/icon?job=nuxeo/nuxeo/master)](https://jenkins.platform.dev.nuxeo.com/job/nuxeo/job/nuxeo/job/master/)
 
-# About the Nuxeo Platform
-
+## About the Nuxeo Platform
 
 The [Nuxeo Platform](http://www.nuxeo.com/products/content-management-platform/) is an open source customizable and extensible content management platform for building business applications. It provides the foundation for developing [document management](http://www.nuxeo.com/solutions/document-management/), [digital asset management](http://www.nuxeo.com/solutions/digital-asset-management/), [case management application](http://www.nuxeo.com/solutions/case-management/) and [knowledge management](http://www.nuxeo.com/solutions/advanced-knowledge-base/). You can easily add features using ready-to-use addons or by extending the platform using its extension point system.
 
 The Nuxeo Platform is developed and supported by Nuxeo, with contributions from the community.
 
-# Sub-Modules Organization
-
-The project is splitted in several sub-modules (listed in dependency order):
-
-* **nuxeo-common**:
-Common utilities
-* **nuxeo-runtime**:
-Container and runtime basic services
-* **nuxeo-core**:
-Document/content management core services
-* **nuxeo-services**:
-Basic services such as file manager, directories, document types
-* **nuxeo-theme**:
-Services related to the theme and theme rendering
-* **nuxeo-jsf**:
-JSF related services
-* **nuxeo-webengine**:
-Services and framework related to WebEngine, the Nuxeo lightweight rendering engine
-* **nuxeo-features**:
-Advanced high-level services, such as audit, imaging, publisher, thumbnails, search
-* **nuxeo-dm**:
-The default Nuxeo Platform application, mostly configuration and UI elements
-* **nuxeo-distribution**:
-This module builds, packages and tests the Nuxeo products.
-
-# Building
-
-Open Nuxeo in [Eclipse Che](https://www.eclipse.org/che/):  
-[![Open in Eclipse Che](http://beta.codenvy.com/factory/resources/codenvy-contribute.svg)](https://beta.codenvy.com/f?id=r3pbh9wv7tjvbws7)
-
-## Requirements
-
-Running the Nuxeo Platform requires Java 11.
-Depending on the features you want to use, you may need some third-party software, such as Libre Office and pdftohtml for document preview or ImageMagick for pictures. The list of third-party software is available in our Admin documentation: [Installing and Setting Up Related Software](http://doc.nuxeo.com/x/zgJc).
+## Installation
 
-Building the Nuxeo Platform requires the following tools:
+Please follow the [Installation](https://doc.nuxeo.com/n/b1j) guide.
 
-* JDK 11 (Oracle's JDK or OpenJDK recommended)
-* Apache Maven 3.1.1+ (3.2+ recommended)
-* Apache Ant 1.7.1+
-* Git (obviously)
+Running the Nuxeo Platform requires Java 11.
```

Comment:
JDK or JRE, is this sync with other docs (I remember there was >=11)

Reply:
https://doc.nuxeo.com/nxdoc/next/installation/
> JDK 11 (Oracle's JDK or OpenJDK)

Yes, let's stick with that, upper versions aren't tested.